src/Model/Photo.php

```diff
@@ -186,26 +189,29 @@ public static function exists(array $conditions)
 	 * @return \Friendica\Object\Image
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 * @throws \ImagickException
+	 * @throws StorageException
 	 */
 	public static function getImageDataForPhoto(array $photo)
 	{
 		if (!empty($photo['data'])) {
 			return $photo['data'];
 		}
 
-		$backendClass = DI::storageManager()->getByName($photo['backend-class'] ?? '');
-		if (empty($backendClass)) {
+		try {
+			$backendClass = DI::storageManager()->getByName($photo['backend-class'] ?? '');
+			$backendRef   = $photo['backend-ref'] ?? '';
+			return $backendClass->get($backendRef);
```

Comment:
Unnecessary intermediate variable.
```suggestion
			return $backendClass->get($photo['backend-ref'] ?? '');
```